openedx/core/djangoapps/team_api/models.py

```diff
@@ -0,0 +1,88 @@
+"""
+Django models related to teams functionality.
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from .utils import slugify
+
+from xmodule_django.models import CourseKeyField
+
+
+class CourseTeam(models.Model):
+    """
+    This model represents team related info.
+    """
+
+    class Meta:
+        unique_together = (('team_id', 'course_id'),)
+
+    team_id = models.CharField(max_length=100, db_index=True)
+    name = models.CharField(max_length=100)
```

Comment:
I'm wondering if we should have a larger max_length (for both ID and name). How did you decide on 100?